src/components/Grid/index.js

```diff
@@ -1,9 +1,15 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
 import GridItem from 'components/GridItem';
 
 import { GridWrapper } from './styles';
 
 class Grid extends Component {
+  static propTypes = {
+    stories: PropTypes.array.isRequired,
```

Comment:
One enhancement is we could use the `arrayOf` type and create a shared `stories` prop like you used in `ListItem`